frontend/src/metabase/visualizations/lib/LineAreaBarRenderer.js

```diff
@@ -103,6 +103,8 @@ function getDatas({ settings, series }, warn) {
           ? HACK_parseTimestamp(row[0], s.data.cols[0].unit, warn)
           : isDimensionNumeric(series)
           ? row[0]
+          : row[0] === null
+          ? "(empty)"
```

Comment:
I think it would be good to pull `"(empty)"` into a `const` somewhere, maybe a `visualizations/lib/format` file.

I was also wondering if we should use a magic string that's very unlikely to appear in the data (`__METABASE_NULL__` or something), but `(empty)` is probably already rare enough.

Reply:
Remember to also mark it as translatable :slightly_smiling_face: